build/imports-generator.js

```diff
@@ -0,0 +1,62 @@
+/* global Promise */
+
+const fs = require('fs');
+const path = require('path');
+const browserify = require('browserify');
+const { dirname: getDirName } = require('path');
+const makeDir = require('make-dir');
+
+const inputFile = path.join(
+	__dirname,
+	'..',
+	'lib',
+	'core',
+	'imports',
+	'index.js'
+);
+const outputFile = path.join(
+	__dirname,
+	'..',
+	'tmp',
+	'core',
+	'imports',
+	'index.js'
+);
+
+async function run() {
+	browserify(inputFile).bundle(async (err, result) => {
+		if (err) {
+			throw new Error('Cannot browserify axe.imports', err);
+		}
+		try {
+			await createFile(outputFile, result);
+		} catch (error) {
+			throw new Error('Cannot write browserify generated axe.imports', error);
+		}
+	});
+}
+
+// exec
+run();
+
+/**
+ * Create file with contents at specified location
+ * @method createFile
+ * @param {String} path file path, inclusive of file name
+ * @param {String} content contents of the file
+ * @returns {Promise}
+ */
+function createFile(path, content) {
```

Comment:
Not a blocker, but I'd still like to see this simplified. Maybe:

```js
const makeDir = require('make-dir')
const fs = require('fs')
const {promisify} = require('util')

const writeFile = promisify(fs.writeFile)

const createFile = (path, content) => makeDir(getDirName(path)).then(() => writeFile(path, content))
```

Reply:
Between, this PR and the axe rule generator PR, there is more or less the same createFile fn. Once both are merged. I plan to refactor this into a shared utility fn.